tests/fsharp/typecheck/sigs/neg69.vsbsl

```diff
@@ -1,3 +1,6 @@
+neg69.fsx(87,6,87,12): typecheck error FS0912: This declaration element is not permitted in an augmentation
```

Comment:
These 2 newly appearing typecheck errors are suspicious.
For other readers, this is the construct defined:
(ivy arg declaration has a superfluos **)** as far as I can see)
```
type Planet(ipx:float<AU>,ipy:float<AU>,
            ivx:float<AU/s>,ivy:float<AU/s)>,
            brush:Brush,mass:float<kg>,
            width,height) =

    // For this sample e store the simulation state directly in the object
    let mutable px = ipx
    let mutable py = ipy
    let mutable vx = ivx
```

Reply:
Yes, it happens because of `)`, and the parser can't handle such errors well either. There're many other errors produced on this type.